docs/components/optimize/userguide/processes.md

```diff
@@ -0,0 +1,31 @@
+---
+id: processes
+title: Processes
+---
+
+
+## Overview
+
+The processes section give you an overview of all the processes and their status on a single page.
+It allows you to set a process owner to take responsibility of the process and time goals to track its performance.
+
+![Processes page](./img/processes.png)
+
+
+
+## Setting an owner: 
+Assign the responsibility for process health to have the right conversation with the right person to fix a process. Once you open up the set owner modal, you can select one of the users in your organization to be the main person responsible for the process.
+
+![Set Owner](./img/set-owner.png)
+
+Once an owner is set on a process, You can sort the processes page by owner to see how each of your process owners perform.
```

Comment:
also just a matter of taste, I think it's not really about owner performance but neat to get a quick overview of processes by owner?
```suggestion
Once an owner is set on a process, You can sort the processes page by owner to quickly see which processes have a common owner.
```